### ISSUE TITLE
Pre tag flow bug

### ISSUE THREAD

**Issue by davidtorosyan** [tnorbye/kdoc-formatter#58]
The `<pre>` tag causes some weirdness with flowing.

Before:
```kt
/**
 * This tag messes things up.
 * <pre>
 *
 * @return some correct
 * value
 */
fun foo() = Unit

/**
 * Even if it's closed.
 * <pre></pre>
 *
 * @return some correct
 * value
 */
fun foo() = Unit

/**
 * This is fine though.
 *
 * @return some correct
 * value
 */
fun foo() = Unit
```

After:
```kt
/**
 * This tag messes things up.
 * <pre>
 *
 * value
 * @return some correct
 */
fun foo() = Unit

/**
 * Even if it's closed.
 * <pre></pre>
 *
 * value
 * @return some correct
 */
fun foo() = Unit

/**
 * This is fine though.
 *
 * @return some correct value
 */
fun foo() = Unit
```

When using default options on v1.5.1.

**Comment by davidtorosyan** [comment]
Would it make sense to fix this by replacing `<pre>` and `</pre>` with triple-backticks?

**Comment by tnorbye** [comment]
This is fixed in 1.5.4 (released just now).

Replacing <pre> with ``` is a good suggestion, because it turns out <pre> and ``` behave slightly differently (and ``` is handled better by Dokka when it isn't terminated for example). We should track that in a separate issue; the fix above fixes the kdoc formatter to gracefully handle unterminated pre blocks (and ``` blocks), as well as fixes an unrelated issue which didn't handle <pre> and </pre> on the same line.
